### ISSUE TITLE
Self-Normalizing Neural Networks

### ISSUE THREAD

**Issue by icoxfog417** [arXivTimes/arXivTimes#328]
## 一言でいうと

通常のフィードフォワードで、伝搬時に正規化の状態をキープし続けられるようにする手法の提案。これにより、Batch Normalizationなど外部的に正規化する必要がなくなる。平均/分散を調整できるよう、正負の値/拡大縮小双方が取れる活性化関数・初期化方法などを提案している。

### 論文リンク

https://arxiv.org/abs/1706.02515

### 著者/所属機関

Günter Klambauer, Thomas Unterthiner, Andreas Mayr, Sepp Hochreiter
Sepp Hochreiter

**Comment by icoxfog417** [comment]
とても分かりやすいまとめ

[Self-Normalizing Neural Networks](https://github.com/kevinzakka/research-paper-notes/blob/master/snn.md)
